For Irene/Stack/0739_Daily_Temperatures.c

```diff
@@ -0,0 +1,26 @@
+
+
+/**
+ * Note: The returned array must be malloced, assume caller calls free().
+ */
+int* dailyTemperatures(int* tem, int temperaturesSize, int* returnSize){
+    int mark = -1;
+    int arrtemp[temperaturesSize];
+    int arrday[temperaturesSize];
```

Comment:
Based on [Google Coding Style for C++](https://google.github.io/styleguide/cppguide.html#Variable_Names), suggest changing the name from arrayday to arrayDay or array_day, so does arrtemp.